src/multi-selector.js

```diff
@@ -47,11 +48,21 @@ angular.module('cp-multi-selector')
 				/**
 				 * Change the items in selectedItems to have correct info whether or not they have a first and last name.
 				 */
-				scope.formatName = function() {
-					return scope.selectedItems.map((item) => {
-						item.formatted_name = (item.first_name && item.last_name) ? `${item.first_name} ${item.last_name}` : item.name;
-						return item;
-					})
+				function formatName() {
+					if(!scope.displayName){
+						if(scope.selectedItems){
+							scope.selectedItems.map((item) => {
```

Comment:
You might consider creating a sub-function for this functionality that takes as its argument an array. Then you can pass either scope.selectedItems or scope.source to it, call .map on whatever one it is, and consolidate this a bit.